src/docker.ts

```diff
@@ -1,7 +1,7 @@
-import {info, warning, startGroup, endGroup} from '@actions/core';
+import {info, debug, warning, startGroup, endGroup} from '@actions/core';
 import {exec} from './exec';
 import {Inputs} from './inputs';
-import {setRegistry} from './state';
```

Comment:
Why change this?

Reply:
I added an `outputs` module in the last PR. This change makes it clear that the set is in the state and not in the outputs. Eg. it is easy to understand what `state.setRegistry()` does without having to look at the imports.

Reply:
It's just to improve readability.